### PR TITLE
Update WebSphere Liberty logos

### PR DESCRIPTION
**Pull request by awisniew90**
Signed-off-by: Adam Wisniewski <awisniew@us.ibm.com>

### What does this PR do?:
Update the WebSphere Liberty stack logos

### Which issue(s) this PR fixes:
N/A

### PR acceptance criteria:

- [ ] Contributing guide
_Have you read the [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) and followed its instructions?_
- [ ] Test automation
_Does this repository's tests pass with your changes?_
- [ ] Documentation
_Does any documentation need to be updated with your changes?_
- [ ] Check Tools Provider
_Have you tested the changes with existing tools, i.e. Odo, Che, Console? (See [devfile registry contributing guide](https://github.com/devfile/registry/blob/main/CONTRIBUTING.md) on how to test changes)_

### How to test changes / Special notes to the reviewer:

### CODE REVIEW

File: stacks/java-websphereliberty-gradle/devfile.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ metadata:
   version: 0.3.1
   displayName: 'WebSphere Liberty Gradle'
   description: Java application Gradle-built stack using the WebSphere Liberty runtime
-  icon: https://raw.githubusercontent.com/OpenLiberty/logos/7fbb132949b9b2589e18c8d5665c1b107028a21d/logomark/svg/OL_logomark.svg
+  icon: https://raw.github.ibm.com/websphere/logos/main/liberty/liberty-was-500-purple.svg?token=AAAJJDABPW7QHG2QWS75L5DCWNSDG
```
Comment (johnmcollier) [inline stacks/java-websphereliberty-gradle/devfile.yaml:23]: Can we link the icons from somewhere other than IBM's GHE? If that token ever expires, then the icon won't be available.

File: stacks/java-websphereliberty/devfile.yaml [hunk 2]
```diff
@@ -20,7 +20,7 @@ metadata:
   version: 0.8.1
   displayName: 'WebSphere Liberty Maven'
   description: Java application Maven-built stack using the WebSphere Liberty runtime
-  icon: https://raw.githubusercontent.com/OpenLiberty/logos/7fbb132949b9b2589e18c8d5665c1b107028a21d/logomark/svg/OL_logomark.svg
+  icon: https://raw.github.ibm.com/websphere/logos/main/liberty/liberty-was-500-purple.svg?token=AAAJJDABPW7QHG2QWS75L5DCWNSDG
```
Comment (johnmcollier) [inline stacks/java-websphereliberty/devfile.yaml:23]: Same as above.

**Review by johnmcollier — CHANGES_REQUESTED** [review]

File: stacks/java-websphereliberty-gradle/devfile.yaml (hunk 1 above)
Comment (awisniew90) [reply stacks/java-websphereliberty-gradle/devfile.yaml:23]: Hi John - agreed. I'm looking to see if there is another location we can reference other that GHE. Keeping the PR draft until then.

Comment (awisniew90) [reply stacks/java-websphereliberty-gradle/devfile.yaml]: @johnmcollier - Links are updated now to a public GH location.

**Review by johnmcollier — APPROVED** [review]

**Pull request merged**